ui/src/components/table/table-element/ColumnSettings.tsx

```diff
@@ -0,0 +1,78 @@
+'use client';
+import { Label } from '@/components/ui/label';
+import { Switch } from '@/components/ui/switch';
+import { useState } from 'react';
+import { GroupingPath } from '@/models/groupings-api-results';
+import { Button } from '@/components/ui/button';
+import {
+    DropdownMenu,
+    DropdownMenuTrigger,
+    DropdownMenuContent,
+    DropdownMenuRadioGroup,
+    DropdownMenuRadioItem
+} from '@/components/ui/dropdown-menu';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSliders } from '@fortawesome/free-solid-svg-icons';
+import {Table} from '@tanstack/table-core';
+
+interface ToggleProps {
+    table: Table<GroupingPath>;
+}
+const ColumnSettings = ({ table } : ToggleProps) => {
+
+    interface ColumnVisibilityState {
+        DESCRIPTION: boolean;
+        'GROUPING PATH': boolean;
+    }
+
+    const [columnVisibility, setColumnVisibility] = useState<ColumnVisibilityState>({
+        DESCRIPTION: true,
+        'GROUPING PATH': true,
+    })
```

Comment:
Could you wrap DESCRIPTION with quotes to make it consistent with 'GROUPING PATH'? Thanks.

Reply:
done